gno.land/cmd/gnoland/secrets.go

```diff
@@ -21,7 +25,7 @@ const (
 )
 
 const (
-	nodeIDKey              = "NodeID"
+	nodeIDKey              = "ID"
 	validatorPrivateKeyKey = "ValidatorPrivateKey"
 	validatorStateKey      = "ValidatorState"
```

Comment:
These are unused

Reply:
We use these IDs for the `secrets init` and `secrets verify` commands, because the secrets are completely separate entities from each other (in contrast to the config, which is a single structure / file)

I have updated these IDs and streamlined their usage:
[3848b21](https://github.com/gnolang/gno/pull/2393/commits/3848b21007f98aa759124a424340a5621a9e3a94)